apps/api/tests/test_cart.py

```
@@ -0,0 +1,365 @@
+class TestCartMerge:
+    """Testes para merge de carrinho (guest -> autenticado)"""
+
+    def test_merge_guest_cart_on_login(self, client):
+        """Deve fazer merge do carrinho de guest ao fazer login"""
+        # 1. Adicionar item como guest
+        guest_item = {
+            'product_id': str(uuid.uuid4()),
+            'quantity': 1
+        }
+
+        guest_response = client.post('/api/cart/items', json=guest_item)
```

Ruff (0.14.5)

Local variable `guest_response` is assigned to but never used

Remove assignment to unused variable `guest_response`

(F841)